src/main/kotlin/de/bund/digitalservice/useid/tracking/matomo/MatomoTrackingService.kt

```diff
@@ -20,26 +21,43 @@ import kotlin.text.Charsets.UTF_8
  * Documentation about matomo events
  * https://matomo.org/guide/reports/event-tracking/
  */
-@Profile("!local")
```

Comment:
Why removing this? We shouldn't try to send events to `example.org` locally imho.